### ISSUE TITLE
[Bug]: Local Ollama models do not work in Windows when OLLMA_HOST is set to 0.0.0.0

### ISSUE THREAD

**Issue by ksylvan** [danielmiessler/fabric#314]
### What happened?

I set OLLAMA_HOST to `0.0.0.0` so when ollama server starts, it binds to localhost and to the WSL interface.

This works correctly and I can see the Ollama server:

```
PS C:\Users\kayvan> Invoke-WebRequest -Uri http://localhost:11434

StatusCode        : 200
StatusDescription : OK
Content           : Ollama is running
RawContent        : HTTP/1.1 200 OK
                    Date: Sun, 31 Mar 2024 03:00:54 GMT
                    Content-Type: text/plain; charset=utf-8
                    Content-Length: 17

                    Ollama is running
```

And in WSL:

```
kayvan@SHAKTI:/mnt/c/Users/kayvan$ ip route
default via 172.23.128.1 dev eth0
172.23.128.0/20 dev eth0 proto kernel scope link src 172.23.128.141
kayvan@SHAKTI:/mnt/c/Users/kayvan$ curl http://172.23.128.1:11434; echo ""
Ollama is running
```

`fabric --listmodels` fails to list local models.

See the discussion in https://github.com/danielmiessler/fabric/discussions/271

### Version check

- [X] Yes I was.

**Comment by ksylvan** [comment]
I have a quick PR. Please assign this to me @danielmiessler

**Comment by ksylvan** [comment]
The above PR fixes this.

**Comment by danielmiessler** [comment]
Added, thank you!

**Comment by ksylvan** [comment]
Hi @danielmiessler this issue isn't closed till PR #315 is merged.
